spotlight-client/src/DataStore/UserStore.ts

```diff
@@ -112,6 +117,10 @@ export default class UserStore {
 
     if (await auth0.isAuthenticated()) {
       const user = await auth0.getUser();
+      const claims = await auth0.getIdTokenClaims();
+      const stateCode = claims[
+        AUTH0_APP_METADATA_KEY
```

Comment:
In order to add the state_code to our auth0 id token claims, I added a custom rule:
<img width="1036" alt="Screen Shot 2022-04-25 at 6 07 58 PM" src="https://user-images.githubusercontent.com/5040739/165198887-8644ec49-e232-4934-8223-3f3e0c7ce955.png">
<img width="910" alt="Screen Shot 2022-04-25 at 6 08 04 PM" src="https://user-images.githubusercontent.com/5040739/165198893-3bb76af6-7322-4f31-9640-9ce43c9bb860.png">

I have to set a namespace with a URL, hence the key

Reply:
Nice!! I think it's best practice to also check in any Auth0 rule you add to Auth0 to the codebase as well in this directory, so it can go through code review / get tracked changes via version control: https://github.com/Recidiviz/public-dashboard/tree/main/spotlight-client/auth0/hooks